README.md

```diff
@@ -24,9 +24,10 @@
 - [Abuse Desk for YOURLS](http://blog.yourls.org/2011/04/yourls-abuse-anti-spam-plugins/) - A Google Safe Browsing implementation for YOURLS to avoid spam links
 - [Access Control Allow Origin](https://github.com/TEODE/yourls-access-control-allow-origin) - Prevents CORS issue with domain CNAMES and aliases for admin actions
 - [Additional Charsets](https://github.com/josheby/yourls-additional-charsets) - Define additional character sets for short URLs
-- [Advanced Reserved URLs](https://github.com/josheby/yourls-advanced-reserved-urls) - Extends the reserved word functionality, blocking short URLs containing reserved words, even if mixed case or written in leetspeak
+- [Admin Issue Diagnostic - A.I.D.](https://github.com/PopVeKind/admin_issue_diagnostic) ❣️ - Collects YOURLS environmentals and formats a template to copy/paste into a YOURLS issue. Just fill in the blanks! Fantastic for newbies, great for experts!
```

Comment:
To second @ozh and to test a [new](https://blog.github.com/changelog/2018-10-16-suggested-changes/) GitHub feature 😉 

```suggestion
- [Admin Issue Diagnostic - A.I.D.](https://github.com/PopVeKind/admin_issue_diagnostic) - Collects YOURLS environmentals and formats a template to copy/paste into a YOURLS issue. Just fill in the blanks! Fantastic for newbies, great for experts!
```

Reply:
@LeoColomb - I don't personally mind deleting the Icon, however this would likely be a disservice to both Newbies and more work for the _YOURLS Team_. I've explained myself below how this plugin has a special status. Plus an alternate suggestion. But I will defer to @ozh decision after he considers my explanation.